src/api/models/sql_models/beltfish.py

```diff
@@ -13,7 +12,20 @@ class BeltFishObsSQLModel(BaseSUSQLModel):
     sql = f"""
         WITH se AS (
             {sample_event_sql_template}
-        )
+        ),
+        pseudosu_su AS MATERIALIZED (
```

Comment:
I did test without it and the CTE worked, but I want to make darn sure it's clear to postgres not to inline the subquery, so that the pseudosu ids are deterministic. (In general this is necessary for the pg 13 upgrade that we're doing as well as django/python.)